File: python/prism_sdk/multimodal_knowledge_representation_inference.py
"""Worldgen P04 AFA-worldgen-P04-F02 product wrapper."""
from .worldgen_knowledge_representation_support import KnowledgeRepresentationRequest, KnowledgeRepresentationReceipt, manifest as _manifest, represent as _run
FEATURE_ID="AFA-worldgen-P04-F02"
CONTRACT_VERSION="worldgen-multimodal-knowledge-representation/1.0"
def worldgen_multimodal_knowledge_representation_inference_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="KnowledgeRepresentationRequest1@1", scale="multimodal multi-study", autonomy_tier="A1")
def represent_worldgen_multimodal_knowledge(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="multimodal multi-study", require_federation=False)
__all__=["KnowledgeRepresentationRequest","KnowledgeRepresentationReceipt","worldgen_multimodal_knowledge_representation_inference_manifest","represent_worldgen_multimodal_knowledge"]

File: crates/worldgen/src/multimodal_knowledge_representation_inference.rs
//! Worldgen P04-F02 multimodal knowledge-representation inference engine.
use super::knowledge_representation_support::{self, KnowledgeRepresentationRequest, KnowledgeRepresentationReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P04-F02"; pub const CONTRACT_VERSION:&str="worldgen-multimodal-knowledge-representation/1.0";
pub fn worldgen_multimodal_knowledge_representation_inference_manifest()->serde_json::Value{knowledge_representation_support::manifest(FEATURE_ID,CONTRACT_VERSION,"KnowledgeRepresentationRequest1@1","multimodal multi-study","A1")}
pub fn represent_worldgen_multimodal_knowledge(request:&KnowledgeRepresentationRequest)->Result<KnowledgeRepresentationReceipt,knowledge_representation_support::KnowledgeRepresentationError>{knowledge_representation_support::represent(request,FEATURE_ID,CONTRACT_VERSION,"multimodal multi-study",false)}
pub use knowledge_representation_support::{KnowledgeRepresentationError,KnowledgeRepresentationReceipt as WorldgenMultimodalKnowledgeRepresentationReceipt,KnowledgeRepresentationRequest as WorldgenMultimodalKnowledgeRepresentationRequest};
